web/src/components/PageSkeleton.tsx

```diff
@@ -0,0 +1,55 @@
+import Box from '@mui/material/Box'
+import Skeleton, { SkeletonProps } from '@mui/material/Skeleton'
+import Stack from '@mui/material/Stack'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement } from 'react'
+
+const StyledSkeletonBreadcrumb = styled(Skeleton)<SkeletonProps>`
+  margin-top: 16px;
+`
+
+const StyledStack = styled(Stack)`
+  width: 100%;
```

Comment:
🔧 No need for a styled component here, you can just directly set the width in the jsx.